config/certs/C=US, O=Exostar LLC, OU=Certification Authorities, CN=Exostar Federated Identity Service Root CA 2 1770471439.pem

```diff
@@ -1,10 +1,10 @@
 Subject: /C=US/O=Exostar LLC/OU=Certification Authorities/CN=Exostar Federated Identity Service Root CA 2
```

Comment:
I noticed similarly that the new file name added a suffix when generating the replacement. In #344 I went out of my way to rename the generated file to keep the original name. Not sure if it's good or bad, but it _worked_, and seems a bit cleaner?

Then again, not sure why this wouldn't be part of the Rake task if that's how we'd want the substitution to happen.

Reply:
I can update it, idk why it added a suffix, maybe because the expiring cert already existed at the time, since i didnt remove it before adding this one

Reply:
yeah, it'll avoid name collisions if the file exists at the time